src/library_glut.js

```diff
@@ -310,22 +336,27 @@ var LibraryGLUT = {
     GLUT.initTime = Date.now();
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
+    if (isTouchDevice) {
+      // Historically handling of touch events was made in completely wrong way.
```

Comment:
Oh sorry, yeah, I realize now I was late to the party and this was already merged. No trouble, pushed https://github.com/kripken/emscripten/commit/7bd0dd2f125c0d2eee246ed455488ca8ce7f775f to refresh the comment a bit. @caiiiycuk: please let me know if that is not accurate, and I can revise.